go/libkb/notify_router.go

```diff
@@ -377,6 +379,31 @@ func (n *NotifyRouter) HandleFSActivity(activity keybase1.FSNotification) {
 	}
 }
 
+// HandleFSPathUpdated is called for any path update notification. It
+// will broadcast the messages to all curious listeners.
+func (n *NotifyRouter) HandleFSPathUpdated(path string) {
+	if n == nil {
+		return
+	}
+	// For all connections we currently have open...
+	n.cm.ApplyAll(func(id ConnectionID, xp rpc.Transporter) bool {
+		// If the connection wants the `Kbfs` notification type
+		if n.getNotificationChannels(id).Kbfs {
+			// In the background do...
+			go func() {
+				// A send of a `FSPathUpdated` RPC with the notification
+				(keybase1.NotifyFSClient{
+					Cli: rpc.NewClient(xp, NewContextifiedErrorUnwrapper(n.G()), nil),
+				}).FSPathUpdated(context.Background(), path)
+			}()
```

Comment:
Any reason not to do:
```go
go (keybase1.NotifyFSClient{Cli: rpc.NewClient(...)}).FSPathUpdated(context.Background(), path)
```

Reply:
I don't see one, but I was just being consistent with the other functions in this file (not written by me).